runsc/sandbox/sandbox.go

```diff
@@ -352,6 +355,13 @@ func (s *Sandbox) createSandboxProcess(conf *boot.Config, args *Args, startSyncF
 	binPath := specutils.ExePath
 	cmd := exec.Command(binPath, conf.ToFlags()...)
 	cmd.SysProcAttr = &syscall.SysProcAttr{}
+	if conf.CoverageFilesDirectory != "" {
+		inCoverGenerate = true
```

Comment:
nit: `inCoverGenerate` implies that we are in the process of generating cover files if it is true. Maybe it should be called something like `generateCoverFiles`.